edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -336,8 +336,9 @@ impl MakeModuleRuntime for DockerModuleRuntime {
                     })
                     .join(notary_registries)
                     .map(move |(client, (notary_registries, _))| {
-                        let mut system_resources = System::new_all();
-                        system_resources.refresh_all();
+                        // to avoid excessive FD usage, we will not allow sysinfo to keep files open.
+                        sysinfo::set_open_files_limit(0);
```

Comment:
I've decided to disable all FD caching. Don't see a point to have just a "small" number there. Let me know if you have other suggestions.

Reply:
0 sounds good to me. Any non-zero value would be completely arbitrary because it would still end up needing to open files for the rest of the processes anyway.